tools/deployed/mainnet.PairMentoV2.addr.json

```diff
@@ -0,0 +1 @@
+{"address":"0xa7cBD42Ee1d9832ac6d15c04Be4E88340dd3951f"}
```

Comment:
this is the same address as baklava, since the contract has not been deployed on mainnet